### PR TITLE
Language 2023-04-01 same as 2022-05-01 and copying from dev to specification

### PR DESCRIPTION
**Pull request by AmirGeorge**
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [ ] a public preview
- [✅ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
* Design Document:
* Previous Open API Doc:
* Updated paths:

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

**Comment by heaths** [comment]
The breaking changes are fine because they are to response-only models in a new api-version. @AmirGeorge can you look at the Avocado errors and fix those? Seems there's a few sample swaggers that aren't referenced. Maybe they are extraneous or were, at some point, refactored into other files.

**Review by heaths — CHANGES_REQUESTED** [review]
Need to fix the Avocado errors, but otherwise fine. The breaking changes are red herrings.

**Review by kristapratico — CHANGES_REQUESTED** [review]
The runtime features for TA/CLU that will not GA need to be removed from the 2023-04-01 specs.

**Comment by heaths** [comment]
Still one Avocado error to resolve:

image

**Comment by wangyuantao** [comment]
Summarization will GA pre-built stills in 2023-04-01 API. I'm making changes to your PR, please on hold. Thanks!

**Comment by wangyuantao** [comment]
> Still one Avocado error to resolve:
> 
> image

Fixed

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 1]
```diff
@@ -3577,31 +2879,31 @@
...
-          "$ref": "#/definitions/DocumentDetectedLanguage"
+          "$ref": "common.json#/definitions/PreBuiltResult"
         }
+      ],
+      "required": [
+        "documents"
       ]
     },
-    "AbstractiveSummarizationResult": {
+    "AbstractiveSummaryDocumentResultWithDetectedLanguage": {
```
Comment (kristapratico) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:2897]: I thought auto language detection would not GA, should this type have the detected language property removed?

File: dev/cognitiveservices/data-plane/Language/questionanswering-authoring.json [hunk 2]
```diff
@@ -1,1804 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (wangyuantao) [reply dev/cognitiveservices/data-plane/Language/questionanswering-authoring.json:1]: QA was not in in 2022-05-01 and won't GA in 2023-04-01, so removed. This PR is to keep 2023-04-01 the same as previous GA version 2022-05-01, and adding new features to be GA in 2023-04-01.

[the remark this answers is not in the document]
Comment (heaths) [reply dev/cognitiveservices/data-plane/Language/questionanswering-authoring.json:1]: That's what was agreed upon in email as well.

**Review by heaths — APPROVED** [review]

File: specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzetext.json [hunk 3]
```diff
@@ -0,0 +1,2975 @@
...
+    },
+    "AbstractiveSummary": {
+      "type": "object",
+      "description": "An object representing a single summary with context for given document.",
+      "properties": {
+        "text": {
+          "type": "string",
+          "description": "The text of the summary."
+        },
+        "contexts": {
```
Comment (deyaaeldeen) [inline specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzetext.json:2943]: I thought you were planning to remove this, are you sure you want to GA it?

**Review by deyaaeldeen — CHANGES_REQUESTED** [review]
Double checking one thing

**Review by kristapratico — APPROVED** [review]

File: specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzetext.json (hunk 3 above)
Comment (wangyuantao) [reply specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzetext.json:2943]: Good catch. Yes, we do considered to remove this, but didn't finish in service, and no timeline to remove this yet. So we prefer to keep it the same as preview API, so preview customers will be easy to migrate.

Comment (deyaaeldeen) [reply specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzetext.json:2943]: Please note that once you GA it you will not be able to remove it later. You may consider removing it before GA and adding it later if customers indeed ask for it.

**Review by deyaaeldeen — APPROVED** [review]

File: specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzetext.json (hunk 3 above)
Comment (deyaaeldeen) [reply specification/cognitiveservices/data-plane/Language/stable/2023-04-01/analyzetext.json:2943]: Discussed offline, keeping it.

**Pull request merged**